src/Microsoft.Health.Fhir.CosmosDb/Features/Search/CosmosSearchService.cs

```diff
@@ -66,18 +62,11 @@ public class CosmosSearchService : SearchService
             SearchOptions searchOptions,
             CancellationToken cancellationToken)
         {
-            string ct = null;
```

Comment:
I don't think it's a problem but this is not backwards compatible. (e.g., if there is a search happening while upgrade is going on, then the continuation token would become invalid and the client would receive error). Consider we don't have real load right now that might be okay but just wanted to call out.

Reply:
That's true.